cmd/controller-manager/app/options/options.go

```diff
@@ -123,6 +128,16 @@ func (o *Options) AddFlags(flags *pflag.FlagSet, allControllers []string, disabl
 	flags.BoolVar(&o.EnablePodPruning, "enable-pod-pruning", false, "Enable pod pruning for pod informer. "+
 		"Enabling this can reduce memory usage of the pod informer, but will disable pod propagation.")
 	o.addKlogFlags(flags)
+
+	flags.BoolVar(&o.PrometheusMetrics, "export-prometheus", true, "Whether to expose metrics through a prometheus endpoint")
+	flags.StringVar(&o.PrometheusAddr, "prometheus-addr", "", "prometheus collector address")
```

Comment:
```suggestion
	flags.StringVar(&o.PrometheusAddr, "prometheus-addr", "", "Prometheus collector address")
```
Capitalize the first letter of the description.